src/DotNet.Status.Web/appsettings.json

```diff
@@ -24,25 +24,25 @@
         {
           "Project": "internal",
           "DefinitionPath": "\\dotnet\\arcade-validation\\dotnet-arcade-validation-official",
-          "Branches": [ "master" ],
+          "Branches": [ "main" ],
           "Assignee": "missymessa"
         },
         {
           "Project": "internal",
           "DefinitionPath": "\\dotnet\\arcade-validation\\dotnet-arcade-validation-for-promotion",
-          "Branches": [ "master" ],
+          "Branches": [ "main" ],
           "Assignee": "missymessa"
         },
         {
           "Project": "internal",
           "DefinitionPath": "\\dotnet-source-indexer\\dotnet-source-indexer CI",
-          "Branches": [ "master" ],
+          "Branches": [ "main" ],
           "Assignee": "alperovi"
         },
         {
           "Project": "internal",
           "DefinitionPath": "\\dotnet\\dotnet-arcade-services\\arcade-services-internal-ci",
-          "Branches": [ "master" ],
+          "Branches": [ "main" ],
```

Comment:
I think this one will be okay, since this is monitoring for Arcade Services and that's the one we're changing here.

Reply:
Thanks a lot for confirming.